recipes/android-ndk/all/conanfile.py

```diff
@@ -286,7 +286,6 @@ def package_info(self):
         self.env_info.CONAN_CMAKE_PROGRAM = cmake_wrapper
 
         toolchain = os.path.join(self.package_folder, "build", "cmake", "android.toolchain.cmake")
-        self.conf_info.prepend("tools.cmake.cmaketoolchain:user_toolchain", toolchain)
```

Comment:
maybe add a comment explaining why we must not set `tools.cmake.cmaketoolchain:user_toolchain` in this recipe.